source/reference/bson-type-comparison-order.txt

```diff
@@ -55,6 +55,17 @@ when comparing a field whose value is a single-element array (e.g. ``[
 ``1`` and ``2``. A comparison of an empty array (e.g. ``[ ]``) treats
 the empty array as less than ``null`` or a missing field.
 
+Objects
+-------
+
```

Comment:
.. versionadded:: 3.4

Reply:
This behaviour wasn't changed at 3.4. Not sure what was the first version the behaviour introduced, but just never documented.

Reply:
Ah, that makes sense.